src/util/rng.h

```diff
@@ -0,0 +1,64 @@
+/*********************                                                        */
+/*! \file rng.h
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Aina Niemetz
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2017 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief A Random Number Generator.
+ **
+ ** A random number generator, implements the xorshift* generator
+ ** (see S. Vigna, An experimental exploration of Marsaglia's xorshift
+ ** generators, scrambled. ACM Trans. Math. Softw. 42(4): 30:1-30:23, 2016).
+ **/
+
+#include "cvc4_private.h"
+
+#ifndef __CVC4__RNG_H
+#define __CVC4__RNG_H
+
+namespace CVC4 {
+
+class RNG
```

Comment:
I suggest changing this to Random or RandomGenerator. At the moment, it reads like a constant or a macro instead of a class. I also suggest renaming the file accordingly.